src/commands/qt_args/q_arg_float.py

```diff
@@ -0,0 +1,50 @@
+from PySide6.QtWidgets import QAbstractSpinBox, QDoubleSpinBox
+from src.commands.qt_args.q_arg_value import QArgValue
+
+
+class QArgFloat(QArgValue):
+    ALLOWED_TYPES = (type(None), float, int)
```

Comment:
hmm since int is allowed here can't we just delete the QArgInt class? Or will it screw up the gui?

Reply:
Python says that values with decimal parts are only floats. For example, 0 is not a float, but 0.0 is a float. So I added int to avoid always writing value with decimal parts.